pipeline-modules/deployment-service/aws/basic-eks-regional.yaml

```diff
@@ -166,7 +166,7 @@ template: |
           {% endfor %}
           # aws s3 rm  s3://{{customer_bucket}} --recursive ;
           {% endif %}
-          rm -f values.yaml;
+          rm values.yaml;
```

Comment:
can you put back the -f? this will eliminate a spurious error from the log

Reply:
Actually can you add the "-f" to any of the modules that have that "rm values.yaml"